docs/manual/hybrid_apps.rst

```diff
@@ -0,0 +1,47 @@
+Hybrid Apps
+===========
+
+.. note::
+
+    Check out the `Hybrid App Sample Project <https://github.com/GodotVR/godot_openxr_vendors/tree/master/samples/hybrid-app-sample>`_
+    for a working demo of hybird app functionality.
+
+Hybrid apps are applications capable of swapping between an immersive 3D mode and a 2D panel mode at runtime.
+
+Project Settings
+----------------
+
+To enable hybrid app functionality, the feature must be enabled in your project settings. The setting can be found in **Project Settings** under the **OpenXR** section.
+The **Hybrid App** setting should be visible if **Advanced Settings** are enabled.
+
+.. image:: img/hybrid_apps/hybrid_apps_project_setting.png
+
+By default, hybrid app functionality is disabled. Selecting either of the other options will enable hybrid app features and determine the app's starting mode.
+
+Switching Modes
+---------------
+
+To switch between immersive and panel app modes in your hybrid app, use the :ref:`switch_mode <class_openxrhybridapp_method_switch_mode>` function.
+In order to pass data between modes, this function accepts a ``String`` that can be accessed in the new app mode via the :ref:`get_launch_data <class_openxrhybridapp_method_get_launch_data>` function.
+Storing the data you'd like to share in a `Dictionary <https://docs.godotengine.org/en/stable/classes/class_dictionary.html>`_
+and converting it to/from `JSON <https://docs.godotengine.org/en/stable/classes/class_json.html>`_ is one way you might utilize this feature.
+
+Here's what it might look like if your app is switching from immersive to panel mode and you have a value you'd like to pass along:
+
+.. code-block:: gdscript
+
+    var my_value = 100 # Value we want to share.
+
+    var my_dictionary = {}
+    my_dictionary["my_key"] = my_value
+
+    var my_string = JSON.stringify(my_dictionary)
+    OpenXRHybridApp.switch_mode(OpenXRHybridApp.HYBRID_MODE_PANEL, my_string)
+
+Then, in your newly launched panel application, you could access ``my_value`` like so:
+
+.. code-block:: gdscript
+
+    var my_string = OpenXRHybridApp.get_launch_data()
+    var my_dictionary = JSON.parse_string(my_string)
+    var my_value = my_dictionary.get("my_key", 0) # Value we wanted to share
```

Comment:
And updating this one too:

```suggestion
    var current_score = my_dictionary.get("current_score", 0) # Value we wanted to share
```

I don't know that "current_score" is the best example, but something that shows that this meant to represent the current game state.

In the sample, it's using the current turkey rotation :-)